nrrd/tests/test_reading.py

```diff
@@ -362,6 +362,41 @@ def test_detached_header_no_filename(self):
                                                         ' path is given'):
                 nrrd.read_data(header, fh)
 
+    def test_invalid_lineskip(self):
+        with open(RAW_NRRD_FILE_PATH, 'rb') as fh:
+            header = nrrd.read_header(fh)
+            np.testing.assert_equal(self.expected_header, header)
+
+            # Set the line skip to be incorrect
+            header['line skip'] = -1
+
```

Comment:
I thought I added this one before, but thanks if I haven't.

Reply:
I think you tested the `byte skip` and `line skip` for all encoding except raw. But that's one benefit of code coverage is for, telling us when something isn't tested.